modulus/utils/capture.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         use_gradscaler: bool = True,
         cuda_graph_warmup: int = 11,
         amp_type: Union[float16, bfloat16] = torch.float16,
+        gradient_clip_norm: float | None = None,
```

Comment:
I did not know you could do `float | None` instead of `Union[float, None]`. Crazy...

Reply:
Opted to use `Optional[float]` instead.